tool/tctl/common/auth_command.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 	"encoding/pem"
 	"fmt"
+	"github.com/gravitational/teleport"
```

Comment:
I realize this is a backport, but this should probably be with the other `gravitational/teleport` imports rather than with the stdlib packages.